mantidproject/mantid
# Solid angle option for ILL SANS

**Description of work.**
This PR adds support for the `Rectangle` option for `SolidAngle` for D22 and D33, changes the default solid angle computation on `SANSILLReduction` to this, and removes a redundant value that was logged when loading SANS data and is no longer useful.
 


**To test:**
- Run SANSILL various system test and confirm they pass


Part of #26906 . 






---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
